Cannot find module 'tslib' in v4.4.0

When i run `npx eslint .` with `eslint-import-resolver-typescritp@4.4.0`, I get this error:
```
npx eslint .

<Project>/api-doc/main/doc-ui.ts
  1:1  error  Resolve error: Cannot find module 'tslib'
Require stack:
- <Project>/node_modules/.pnpm/eslint-import-context@0.1.4_unrs-resolver@1.7.2/node_modules/eslint-import-context/lib/index.js
- <Project>/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.0_eslint-plugin-import@2.31.0_eslint@9.27.0_jiti@2.4.2_/node_modules/eslint-import-resolver-typescript/lib/index.cjs
- <Project>/node_modules/.pnpm/eslint-module-utils@2.12.0_@typescript-eslint+parser@8.32.1_eslint@9.27.0_jiti@2.4.2__t_5b76abf6ac0eb8e72fce4ff04c6165ef/node_modules/eslint-module-utils/resolve.js
- <Project>/node_modules/.pnpm/eslint-plugin-import@2.31.0_@typescript-eslint+parser@8.32.1_eslint@9.27.0_jiti@2.4.2___00e39fa7207ca6dcb8d55c3e91e142ca/node_modules/eslint-plugin-import/lib/rules/no-unresolved.js
- <Project>/node_modules/.pnpm/eslint-plugin-import@2.31.0_@typescript-eslint+parser@8.32.1_eslint@9.27.0_jiti@2.4.2___00e39fa7207ca6dcb8d55c3e91e142ca/node_modules/eslint-plugin-import/lib/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1405:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1061:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1066:22)
    at Module._load (node:internal/modules/cjs/loader:1215:37)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Module.require (node:internal/modules/cjs/loader:1491:12)
    at require (node:internal/modules/helpers:135:16)
    at Object.<anonymous> (<Project>/node_modules/.pnpm/eslint-import-context@0.1.4_unrs-resolver@1.7.2/node_modules/eslint-import-context/lib/index.js:4:17)
    at Module._compile (node:internal/modules/cjs/loader:1734:14)  import/no-internal-modules
  1:1  error  Resolve error: Cannot find module 'tslib'
Require stack:
- <Project>/node_modules/.pnpm/eslint-import-context@0.1.4_unrs-resolver@1.7.2/node_modules/eslint-import-context/lib/index.js
- <Project>/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.0_eslint-plugin-import@2.31.0_eslint@9.27.0_jiti@2.4.2_/node_modules/eslint-import-resolver-typescript/lib/index.cjs
- <Project>/node_modules/.pnpm/eslint-module-utils@2.12.0_@typescript-eslint+parser@8.32.1_eslint@9.27.0_jiti@2.4.2__t_5b76abf6ac0eb8e72fce4ff04c6165ef/node_modules/eslint-module-utils/resolve.js
- <Project>/node_modules/.pnpm/eslint-plugin-import@2.31.0_@typescript-eslint+parser@8.32.1_eslint@9.27.0_jiti@2.4.2___00e39fa7207ca6dcb8d55c3e91e142ca/node_modules/eslint-plugin-import/lib/rules/no-unresolved.js
- <Project>/node_modules/.pnpm/eslint-plugin-import@2.31.0_@typescript-eslint+parser@8.32.1_eslint@9.27.0_jiti@2.4.2___00e39fa7207ca6dcb8d55c3e91e142ca/node_modules/eslint-plugin-import/lib/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1405:15)
    at defaultResolveImpl (node:internal/modules/cjs/loader:1061:19)
    at resolveForCJSWithHooks (node:internal/modules/cjs/loader:1066:22)
    at Module._load (node:internal/modules/cjs/loader:1215:37)
    at TracingChannel.traceSync (node:diagnostics_channel:322:14)
    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)
    at Module.require (node:internal/modules/cjs/loader:1491:12)
    at require (node:internal/modules/helpers:135:16)
    at Object.<anonymous> (<Project>/node_modules/.pnpm/eslint-import-context@0.1.4_unrs-resolver@1.7.2/node_modules/eslint-import-context/lib/index.js:4:17)
    at Module._compile (node:internal/modules/cjs/loader:1734:14)  import/no-extraneous-dependencies
```

Downgrading to v4.3.5 works fine.

Missing dependency in v4.4.0?

--- a/.changeset/small-drinks-wave.md
+++ b/.changeset/small-drinks-wave.md
@@ -1,0 +1,5 @@
+---
+"eslint-import-resolver-typescript": patch
+---
+
+fix(deps): bump `eslint-import-context` to v0.1.5

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -17,6 +17,7 @@ jobs:
           - 18
           - 20
           - 22
+          - 24
         os:
           - ubuntu-latest
           - windows-latest
@@ -43,6 +44,8 @@ jobs:
           PARSER_NO_WATCH: true
 
       - name: Codecov
+        # bad Windows -- https://github.com/codecov/codecov-action/issues/1787
+        if: ${{ !github.event.pull_request.head.repo.fork && matrix.os != 'windows-latest' }}
         uses: codecov/codecov-action@18283e04ce6e62d37312384ff67231eb8fd56d24 # v5.4.3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}

--- a/package.json
+++ b/package.json
@@ -70,11 +70,11 @@
   },
   "dependencies": {
     "debug": "^4.4.1",
-    "eslint-import-context": "^0.1.3",
+    "eslint-import-context": "^0.1.5",
     "get-tsconfig": "^4.10.1",
     "is-bun-module": "^2.0.0",
     "stable-hash": "^0.0.5",
-    "tinyglobby": "^0.2.13",
+    "tinyglobby": "^0.2.14",
     "unrs-resolver": "^1.7.2"
   },
   "devDependencies": {
@@ -95,9 +95,9 @@
     "dummy.js": "link:dummy.js",
     "eslint": "^9.27.0",
     "eslint-import-resolver-typescript": "workspace:*",
-    "eslint-plugin-import-x": "^4.13.0",
+    "eslint-plugin-import-x": "^4.13.1",
     "nano-staged": "^0.8.0",
-    "npm-run-all2": "^8.0.3",
+    "npm-run-all2": "^8.0.4",
     "path-serializer": "^0.4.0",
     "premove": "^4.0.0",
     "prettier": "^3.5.3",

